### ISSUE TITLE
Problem with UTF-8 encoding

### ISSUE THREAD

**Issue by munfor** [Adldap2/Adldap2-Laravel#322]
Hi @stevebauman !

Some users in my Active Directory have 'ñ' character in his username.
The problem is when the username is get from the server:

`$request->server($auth[$key])`

https://github.com/Adldap2/Adldap2-Laravel/blob/6c7a671df1c7e4ba412ba01c6a6b60d4fa1994ee/src/Middleware/WindowsAuthenticate.php#L53

It returns 'ñ' like strange symbol. I think that it is a codification problem. Maybe a server configuration problem?

In anyway, encoding to UTF-8 it works for me:

```
if ($account = $request->server($auth[$key])) {
      $account = utf8_encode($account);
      // ...
}
```

Thank you!

> Operating System: Windows Server 2008
> PHP Version: **5.6.8**
> Lavarel Version: **5.4.23**
> Adldap2-Laravel Version:  **2.1**

**Comment by munfor** [comment]
Thank you!
Woow, that is fast !

**Comment by stevebauman** [comment]
Hi @munfor, thanks for letting me know! No problem at all :).

However, this fix will be released in `v3.1` since it will come with a new configuration option (see: [link](https://github.com/Adldap2/Adldap2-Laravel/blob/master/src/Config/auth.php#L126-L160)).

This will be released in the coming days. Just need to finish the documentation for this option.
